closing-channels/closing-channels.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import "fmt"
+
+// Closing a channel indicates that no more values will be
+// sent on it. This is useful to communicate completion to
+// the channel's receivers.
+
+func main() {
+	jobs := make(chan int, 5)
+	done := make(chan bool)
```

Comment:
For channels that only act to indicate that a job is done, you'll often see `chan struct{}` instead of `chan bool`. The reason is super nit-picky (`struct{}{}` uses 0 bytes, but `true` or `false` requires 64 bits (on modern computers), i.e. 8 bytes), but it has become almost a convention.